src/index.js

```diff
@@ -49,8 +49,9 @@ const setHeaders = (headers: Headers) => {
 export {
   DAO_ENTITIES,
   FILTER_OPERATORS,
-}
+};
 
+/* eslint-disable sorting/sort-object-props */
```

Comment:
added ignore here, because eslint rule ignores WordCase here..
there is the options about it in the plugin , though: 
` "sorting/sort-object-props": [  2,  {      "ignoreCase": true   }    ],`
but if I change it to false, it starts throwing errors when `_` symbol is before any letter(like `beverage_srm` before `beverages`). These errors can not be fixed with my sort-plugins in any text-editor automatically, but edit all the cases manually is errors prone :/